index.html

```diff
@@ -230,7 +232,73 @@ <h1>Skills</h1>
           </ul>
         </div>
       </section>
-   
+      <section class="details">
+
+        <div class="contact-p">
+             <p>
+              I'm always interested in hearing about new projects, so if you'd like
+              to chat please get in touch.
+            </p>
+        </div>
+        <form action="https://formspree.io/f/xyyvjvdn" method="POST">
+          <div class="ContactForm">
+              <div class="inputText">
+                      <input type="text" name="name" placeholder="Full name"
+                       required="" maxlength="30" type="text" class="input_t">
+                       <div class="line-"></div>
+                </div>
```

Comment:
[OPTIONAL] 👍🏽 
- Good Job adding the required attribute to all input fields of your form. I'd like to kindly suggest that you replace `required=""` with just `required` since there are no values assigned to it. 

What do you think?? `required=""` works as well, but just for readability 🍒